<?xml version="1.0"?>


<launch>
	<!--<param name="/use_sim_time" value="true"/>-->
	<!-- utm_odom data-->
	
	<!--<node pkg="gps_common" type="utm_odometry_node" name="gps_path_odom" output="screen">
	</node>-->

	<!-- gps_odom data-->
    	<node pkg="gps_common" type="utm_odometry_to_navsatfix_node" name="utm_odometry_node" output="screen">
	</node>
	

	<!-- Pure persuit-->
    	<node pkg="hypharos_minicar" type="Pure_Pursuit" name="get_pure_pursuit" output="screen">
		<remap from="pure_pursuit/odom" to="/odom/gps" />
	</node>


	<!-- Current odom-->
    	<!--<node pkg="gps_common" type="utm_odometry_node" name="gps_odom" output="screen">
	</node>-->
	

	<!-- GPS_Path data-->
    	<node pkg="stauto_sensor" type="gps_path.py" name="create_gps_path" output="screen">
		<remap from="/gps/fix" to="fix" />
		<remap from="/path" to="/pure_pursuit/global_planner" />
	</node>


</launch>
